cli/cmd/connect.go

```diff
@@ -44,25 +46,41 @@ func NewConnectCmd() *cobra.Command {
 	return connectCmd
 }
 
+// isURI returns true if a string is a valid URI.
+func isURI(str string) bool {
+	uriRe := regexp.MustCompile("^((\\w+):(\\/+)?)?([\\w.-]+):(\\d+)$")
```

Comment:
Why it was impossible to use only one regexp?

Reply:
I thought it would be too complicated.

But it doesn't look too scary. Thank you, updated.